### DISCUSSION TITLE
Visual Studio 2022 Stuck when open copilot setting panel without account auth

### DISCUSSION THREAD

**Discussion by rainbowflesh** [community/community#13491] [Copilot Conversations] [answered]
![](https://s3.bmp.ovh/imgs/2022/03/f7cfc1d2347fc622.png)

Open setting panel then chose "GitHub", dialog button got no response, can't close dialog or Visual Studio.
After auth the device via open any code file and follow auth step will get the setting panel work back.

**Comment by Suvraneel** [comment]
@isNijikawa Not replicating in my case
Other people can still verify though...

**Comment by kzu** [comment]
I can reproduce it. It's the same underlying issue as https://github.com/github/feedback/discussions/13883, which also locks up VS. Investigating

**Comment by kzu** [comment, marked as answer]
Please update to the latest version. It contains a fix for this issue. Thanks for reporting it!
